style.css

```diff
@@ -0,0 +1,89 @@
+body {
+        background-image :url("background.png");
```

Comment:
alright; another rule of thumb. Use double quotes in HTML and JSX (do not worry about what JSX is yet, just don't want to contradict myself later :P) and use single quotes everywhere else!